recipe/meta.yaml

```diff
@@ -61,6 +61,7 @@ outputs:
         - curl-config --protocols      # [not win]
         - test -f ${PREFIX}/lib/libcurl${SHLIB_EXT}  # [not win]
         - test ! -f ${PREFIX}/lib/libcurl.a          # [not win]
+        - test ! -f %LIBRARY_BIN%\curl.exe  # [win]
```

Comment:
Because of #81 we already knew that `curl.exe` was there but it is not listed in the `files` section of the output and removing it manually, like in #81, will also remove it from `curl` itself, so that is a no-no.

@msarahan you are the expert on `outputs`, what can we do here?

See https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=215692&view=logs&j=a70f640f-cc53-5cd3-6cdc-236a1aa90802&t=f5d15007-a01c-5ad8-c9ce-4d519d3b275f&l=1928